### DISCUSSION TITLE
Support license detection for SBOM scan

### DISCUSSION THREAD

**Discussion by bedla** [aquasecurity/trivy#6073] [Ideas]
### Description

Hi, 
I have create PR https://github.com/aquasecurity/trivy/pull/6072 for missing License scan of SBOM files.
I am not sure if I am missing something, but it looks like that it might be trivial change. It if make sense to you.
What do you think?
Thx
Ivos

### Target

SBOM

### Scanner

License

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/6257 to track
